routes/delete-project.js

```diff
@@ -27,6 +27,10 @@ module.exports = function(config) {
         return;
       }
 
+      if(req.session.project && req.session.project.meta.id == projectId) {
```

Comment:
Do you think it's necessary to also add a check for `req.session.project.meta` before you try to get `req.session.project.meta.id`?